documentation/docs/pages/operators/nodes/preliminary-steps/vps-setup.mdx

```diff
@@ -35,15 +42,15 @@ Bear in mind that credential signing is primarily CPU-bound, so choose the faste
 
 #### Minimum Requirements
 
-| Hardware | **Minimum Specification**                      |
+| Hardware | **Minimum Specification**                  |
```

Comment:
Do we want to update these now or are they good for the moment?

Reply:
These are good for the moment, we are in process of getting new ones yet